### PR TITLE
Add a note on 'Where to follow new updates'

### PR DESCRIPTION
**Pull request by axonasif**
## Description

We had two community members asking about this recently, so thought to document it here.
- https://discord.com/channels/816244985187008514/1130414457718329354
- https://discord.com/channels/816244985187008514/839379835662368768/1142062097052733530

## Related Issue(s)

Fixes #

## Documentation

/hold

### CODE REVIEW

**Comment by laushinka** [comment]
Thank you, @axonasif!

**Comment by laushinka** [comment]
LGTM, but let me pull in @filiptronicek as well to confirm.

File: README.md [hunk 1]
```diff
@@ -93,3 +93,10 @@ These images are no longer being published:
 You can follow the detailed guide on how to contribute [here](CONTRIBUTING.md).
 
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gitpod-io/workspace-images)
+
+## Where to follow new updates
+
+- **Announcements**: Significant changes and breaking updates are announced through our [Changelog](https://www.gitpod.io/changelog/) with details.
```
Comment (filiptronicek) [inline README.md:99]: Can you please change this to link to our `CHANGELOG.md`? We decided to post updates there instead.

**Review by filiptronicek — APPROVED** [review]
thx <3

File: README.md (hunk 1 above)
Comment (axonasif) [reply README.md]: Thanks! Didn't know.

**Review by filiptronicek — APPROVED** [review]

**Pull request merged**